initial_setup.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+URL="http://0.0.0.0:8000/"
+status_code=000
+ENV_EXAMPLE=".env.example"
+ENV_ORIG=".env"
+#TIMEOUT_TIME=60
+
+exit_on_error () {
+    echo "FAILED AT: $@"
+    exit
+}
+
+# instantiate .env file
+cp ${ENV_EXAMPLE} ${ENV_ORIG}
+
+# runs docker compose as daemon process
+docker-compose up -d || exit_on_error "docker-compose up -d"
+
+#while [[ ${status_code} -ne 302 && ${SECONDS} -lt ${TIMEOUT_TIME} ]];
```

Comment:
When I suggested disabling the timeout, I actually meant to suggest not to wait for the HTTP server to start serving. I think you can start running migrations and the install script much earlier.

Reply:
I cannot run migrations and install scripts until Django and all other requirements have been installed from the `requirements.txt` and `requirements-dev.txt` in the container.
According to my understanding, Django will not take much time to start but the installation of dependencies does. That's why I am waiting for Django to start the HTTP to ensure that the container is up and running with all dependencies installed.

Reply:
You might consider something like [pip3 -vvv freeze -r requirements.txt | grep "not installed"](https://stackoverflow.com/a/64949020) to check if pip dependencies have been installed.

If that doesn't work, checking the HTTP service is acceptable. Django server takes a long time to get ready on my machine, that's why I'm looking for a closer approximation of when the container is ready for migrations.